src/enums/ClaimConditionType.ts

```diff
@@ -0,0 +1,27 @@
+enum ClaimConditionType {
+  StartTime,
+  LimitPerWallet,
+  MerkleRoot,
+  WaitTimeBetweenClaims,
+}
+
+/**
+ * Questions:
+ *
+ * 1. Can there only be a single `startTimestamp` condition? What is a `startTimestamp` condition?
```

Comment:
startTimestamp = seconds into the future when the mint condition will be active.

startTimestamp = 0 -> the condition will be active NOW.
startTimestamp = 60 -> the condition will be active in 60 seconds.

Reply:
0 = start now